src/style/theme.scss

```diff
@@ -132,6 +133,7 @@ $iui-color-background-disabled--dark: $iui-color-background-3--dark;
 
 // Dark theme > Foreground
 $iui-color-foreground-body--dark: #FFF;
+$iui-color-foreground-body-invert--dark: #000;
```

Comment:
Not really sure about this one. I think dark tooltip (with light text) looks fine on both themes, but light tooltip looks out of place in dark theme.